### PR TITLE
Open the Modify Row and Copy button to a new tab

### PR DESCRIPTION
**Pull request by shucon**
### Description

Please describe your pull request.

Fixes # https://github.com/phpmyadmin/phpmyadmin/issues/15413

Before submitting pull request, please review the following checklist:

- [ ] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [ ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [ ] Every commit has a descriptive commit message.
- [ ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

File: libraries/classes/InsertEdit.php [hunk 1]
```diff
@@ -77,12 +77,18 @@ public function getFormParametersForInsertForm(
         array $where_clause_array,
         $err_url
     ) {
+        $query = null;
+        if (isset($_POST['sql_query'])) {
+            $query = $_POST['sql_query'];
+        } else if (isset($_GET['sql_query'])) {
```
Comment (williamdes) [inline libraries/classes/InsertEdit.php:83]: You need to use the `sql_signature` method that is used in some parts of the code base to make this method secure

File: libraries/classes/Util.php [hunk 2]
```diff
@@ -817,7 +817,7 @@ public static function localisedDate($timestamp = -1, $format = '')
 
         return $ret;
     }
-
+
```
Comment (williamdes) [inline libraries/classes/Util.php:820]:
```suggestion
```

**Review by MauricioFauth — COMMENTED** [review]
I'm not sure if I agree with this change. We are moving away from having SQLs in the URL, so I'm not comfortable with this.

**Review by williamdes — CHANGES_REQUESTED** [review]
I also am not too much a fan of this, some users have reported privacy concerns about private data in URLs

**Pull request closed without merging**